docs/product-getting-started-guide/src/main/asciidoc/test-speed-scenarios-con.adoc

```diff
@@ -0,0 +1,11 @@
+[id='test_scenarios_con']
```

Comment:
Fix anchor ID (dashes, not underscores).

Also, I thought test scenarios were part of Tech Preview features, i.e., weren't actually supported in BRMS/DM, but just there? I'm assuming that since Ivo signed off, I'm mistaken, but that's what I last recall. Whatever, I guess.

Reply:
This is what Ivo wants.